content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -24,15 +24,32 @@ This provides observability for libraries not covered by Datadog SDKs without ch
 
 Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 
+Datadog SDKs provide an implementation of the OpenTelemetry API that submits spans to a Datadog Agent. To use the OpenTelemetry
+instrumentations with Datadog SDKs, ensure a Datadog Agent is [running][16].
+
 <div class="alert alert-warning">
 When replacing a Datadog instrumentation with its OpenTelemetry equivalent, disable the
 Datadog instrumentation to avoid duplicate spans in the trace.
 </div>
 
+<div class="alert alert-warning">
+Datadog SDKs implement the OpenTelemetry API by overriding the default implementations in the OpenTelemetry SDK. Operations only supported by the
+OpenTelemetry SDK are not supported (ex: SpanProcessors, OTLP Trace Exporters).
+</div>
+
+<div class="alert alert-warning">
+Datadog SDKs do not support OpenTelemetry Metrics and Logs APIs. To use OpenTelemetry Logs and Metrics APIs, use [OTLP Ingest][12].
+</div>
+
 <div class="alert alert-info">
-<code>DD_TRACE_OTEL_ENABLED</code> is not required for the Datadog Go SDK.
+<code>DD_TRACE_OTEL_ENABLED</code> is not required for the Datadog Go and Ruby SDKs.
 </div>
 
+
+## Configuration
+
+Datadog SDKs support configuration through [Opentelemetry environment variables][16]. 
+
```

Comment:
Re: My comment on the Prerequisites section, I think we can move some content to the Language support section like this:

```
## Language support

Datadog SDKs implement the OpenTelemetry API by overriding the default implementations in the OpenTelemetry SDK. However, note the following limitations:

- Operations only supported by the OpenTelemetry SDK are not supported (for example, SpanProcessors or OTLP Trace Exporters).
- Datadog SDKs do not support OpenTelemetry Metrics and Logs APIs. To use OpenTelemetry Logs and Metrics APIs, use [OTLP Ingest][12].
```